### DISCUSSION TITLE
Help with loanding the aircraft.  Thanks

### DISCUSSION THREAD

**Discussion by darcy1968** [Stratosphere-Studios/777-300ER#88] [General]
As I load the aircraft in Xpl12 a windows apperared with this text:
_There was a minor problme loading the aircrfaft:
Aircraft/777-300ER-master/777.acf
was unable to load the object file "Aircraft/777.300ER-master/objects/fuse_details.obj" that shoud be attacched to this airplane.  The file could be missing or unreadeable.
You may need to update aercraafct o report to the author of the aircraft_"
After I click in Undestood, another windows appears mentiones about Fmod (I read somewhere to dismiss this advice).

Sorry I see its an awesome aircraft but I cannot fly it.   Even in the FMC I cannot start entering data.  Don't it is because of the same issues.

Thank you very much in advance!
And congratulations for this great job!

**Comment by remenkemi** [comment]
This is a known issue, we're waiting on @nathroxer to fix something in that file.

If you click "Understood" it will work fine.

**Comment by darcy1968** [comment]
Thank you for reply!!  Awesome service.
Anyway I click in understood the FMC button, top one at right sise if FMC,
do nothing.  It’s doesn’t work yet?
Thank you very much for your Kinder attention!!
Regards

El El mar, 13 de jun. de 2023 a la(s) 11:35, crazytimtimtim <
***@***.***> escribió:

> This is a known issue, we're waiting on @nathroxer
> <https://github.com/nathroxer> to fix something in that file.
>
> If you click "Understood" it will work fine.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Stratosphere-Studios/777-300ER/discussions/88#discussioncomment-6164449>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AXSHTMT6ZA2WWTECKZXJPJLXLB3DFANCNFSM6AAAAAAZELEKQE>
> .
> You are receiving this because you authored the thread.Message ID:
> <Stratosphere-Studios/777-300ER/repo-discussions/88/comments/6164449@
> github.com>
>
-- 
Enviado con Gmail Mobile

**Reply by remenkemi** [reply]
Thanks! Sorry about the late response

What do you mean click Understood on the FMC? Understood is only on the X-Plane popup, then you have to put in a code in the FMS to unlock it. Just use the backup code to make things easier, it's "BIRDS". Enter that into the bottom left on the FMC and then you will be able to go to the menu using the button on the bottom right!

**Comment by darcy1968** [comment]
Thank you very much for your reply.
I attached a picture for you to see.   I first unlock with "BIRDS" then I
click on the FMC button (as in picture) and nothing happens.  If  I click
on any other  button a message appears: "Key/function inop".
Thanks once again for your support (it is not usual).
Regards

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Reply by remenkemi** [reply]
I can't see the picture you attached. It looks like you are replying by email, but can you try doing it directly on github.com? You should be able to drag the picture into the text box and it will be included in the reply.
Thanks!

**Comment by darcy1968** [comment]
I'll try again.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Libre
de virus.www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
